app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java

```diff
@@ -605,7 +607,12 @@ public void onTextPrompt(GeckoSession session, String title, String msg, String
 
     @Override
     public void onAuthPrompt(GeckoSession session, String title, String msg, AuthOptions options, AuthCallback callback) {
-
+        mAuthPrompt = new AuthPromptWidget(getContext());
+        mAuthPrompt.mWidgetPlacement.parentHandle = getHandle();
+        mAuthPrompt.setTitle(title);
+        mAuthPrompt.setMessage(msg);
```

Comment:
from @bluemarvin in https://github.com/MozillaReality/FirefoxReality/pull/985#issuecomment-470738545:

> I might look into why GV is returning a truncated string, but I noticed most other browsers don't even display the message. I wonder if this is for security reasons.

ICYMI:

- _(I know you both are aware, but for posterity's sake:)_ @MortimerGoro filed [Gecko bug 1533468](https://bugzilla.mozilla.org/show_bug.cgi?id=1533468) to track this GeckoView bug.
- I did some, albeit, limited testing and reported my findings in the code path in the [GeckoView platform bug 1533468](https://bugzilla.mozilla.org/show_bug.cgi?id=1533468#c2).

Reply:
this looks shippable. :+1:

some further findings, for posterity's sake:

I attempted to do some testing to see how the UI appears vs. how it now appears in Firefox Reality in this branch:

## Firefox Reality

![image](https://user-images.githubusercontent.com/203725/54192629-ae70c800-4475-11e9-867f-398230d4e49e.png)

## [Oculus Browser](https://www.oculus.com/experiences/go/1257988667656584/)

Basic-Auth does not appear to be implemented correctly. _(cc: @digitec from the Oculus Browser team.)_

![image](https://user-images.githubusercontent.com/203725/54191863-fabb0880-4473-11e9-87d0-4126938bb4cd.png)

![image](https://user-images.githubusercontent.com/203725/54191868-fc84cc00-4473-11e9-9998-73ae4d5245f3.png)

## [Samsung Internet](https://www.oculus.com/experiences/go/849609821813454/)

Auto-crashes upon launch. (I reported the issue here: SamsungInternet/support#83.)